.rubocop.yml

```diff
@@ -121,22 +292,22 @@ Layout/SpaceInsideBlockBraces:
 Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
 
+# enforces that parentheses do not have spaces
 Layout/SpaceInsideParens:
   Enabled: true
 
-# Check quotes usage according to lint rule below.
+# Enforce string literals to use double quotes everywhere
 Style/StringLiterals:
-  Enabled: false
+  Enabled: true
   EnforcedStyle: double_quotes
 
-# Detect hard tabs, no hard tabs.
-Layout/IndentationStyle:
+# Requires a single final new line to the file
+Layout/TrailingEmptyLines:
   Enabled: true
+  EnforcedStyle: final_newline
 
-# This cop looks for trailing blank lines and a final newline in the source code.
-# This is being disabled because the requirement to have a final newline is illogical.
-Layout/TrailingEmptyLines:
-  Enabled: false
+Layout/EmptyLines:
+  Enabled: true
```

Comment:
Not sure what's the difference between this cop and the last cop but I guess I captured my concern in the last comment.